### DISCUSSION TITLE
Is milvus support vector calc for the result of IVF_PQ topK?

### DISCUSSION THREAD

**Discussion by fredqfc** [milvus-io/milvus#29167] [Q&A and General discussion]
Because IVF_PQ will loss some precision. So I want to calc IP for the topK result

**Comment by yhmo** [comment]
I think it is not possible in current versions.

**Comment by xiaofan-luan** [comment]
you will have to use IVF FLAT or disk Index because IVFPQ don't reserve original vectors
